src/global/install.rs

```diff
@@ -106,6 +106,7 @@ pub(crate) async fn create_executable_trampolines(
     let mut activation_variables = prefix.run_activation().await?;
     let path_after_activation = activation_variables
         .remove("PATH")
+        .or_else(|| activation_variables.remove("Path"))
```

Comment:
Env vars are case insensitive on Windows. It might make sense to make them all uppercase?

Reply:
You mean that the keys of the hashmap that this function returned should be turned to uppercase on Windows?

```rust
    pub fn run_activation(
        &self,
        variables: ActivationVariables,
        environment: Option<HashMap<&OsStr, &OsStr>>,
    ) -> Result<HashMap<String, String>, ActivationError> {
```

Reply:
yep! Or should use a string type that does case insensitive comparisons.